uk.co.compucorp.civicrm.hrcore/templates/CRM/HRCore/Page/Error404Checker.tpl

```diff
@@ -0,0 +1,8 @@
+<p>
+  The page you are looking for can't be found.  Maybe you could try:
+</p>
+<ul>
+  {foreach from=$urls item="currentUrl" key="URLLabel"}
+    <li><a href="{$currentUrl}">{$URLLabel}</a></li>
+  {/foreach}
+</ul>
```

Comment:
Missing newline at the of the file. Please check if your IDE/Editor has an option to add this newline automatically